const.go

```diff
@@ -536,6 +536,7 @@ const (
 	CKM_ECMQV_DERIVE                     = 0x00001052
 	CKM_ECDH_AES_KEY_WRAP                = 0x00001053
 	CKM_RSA_AES_KEY_WRAP                 = 0x00001054
+	CKM_EDDSA                            = 0x00001057
```

Comment:
is there a link for me to verify this number?

Reply:
https://docs.oasis-open.org/pkcs11/pkcs11-curr/v3.0/csprd01/pkcs11-curr-v3.0-csprd01.pdf

Reply:
why do we need to add this manually instead of updating pkcs11.h and regenning the go code?

Reply:
I'm open to either way, but if we go down the regen approach then there needs to be documented steps on how to do so, such as in a Makefile.

Reply:
I'm generating it now directly with go generate, see #147 
Also opened #146 to upgrade wholesale to v3 of pkcs11 spec and then regen, seems easier than adding const piecemeal